content/concurrency.md

```diff
@@ -1,14 +1,60 @@
 ## VIII. Concurrency
 ### Scale out via the process model
 
-Any computer program, once run, is represented by one or more processes.  Web apps have taken a variety of process-execution forms.  For example, PHP processes run as child processes of Apache, started on demand as needed by request volume.  Java processes take the opposite approach, with the JVM providing one massive uberprocess that reserves a large block of system resources (CPU and memory) on startup, with concurrency managed internally via threads.  In both cases, the running process(es) are only minimally visible to the developers of the app.
+#### 1. Any computer program, once run, is represented by one or more processes
+
+Any computer program, once run, is represented by one or more processes. Web
```

Comment:
Do we want to mention the counterexample of loading many applications into one process (e.g. JBoss / WebLogic / WebSphere), or only focus on positive examples?